Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/README.md

```diff
@@ -1,386 +1,218 @@
+    | Public Folder | False |
+    | Has impersonation rights | False |
+    | Use system proxy settings | False |
+    | Fetch incidents | False |
+    | First fetch timestamp (&lt;number&gt; &lt;time unit&gt;, e.g., 12 hours, 7 days) | False |
+    | Mark fetched emails as read | False |
+    | Incident type | False |
+    | ┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉<br/>‎                                           Manual Mode<br/>Exchange Server Hostname or IP address | False |
+    | DOMAIN\USERNAME (e.g. DEMISTO.INT\admin) | False |
+    | Exchange Server Version (On-Premise only. Supported versions: 2007, 2010, 2010_SP2, 2013, and 2016) | False |
+    | Trust any certificate (not secure) | False |
+    | ┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉┉<br/>‎                                         Advanced Mode<br/>Override Authentication Type (NTLM, Basic, or Digest). | False |
+    | Timeout (in seconds) for HTTP requests to Exchange Server | False |
+    | Max incidents per fetch | False |
+    | Run as a separate process (protects against memory depletion) | False |
+
+4. Click **Test** to validate the URLs, token, and connection.
+
+## Fetched Incidents Data
+The integration imports email messages from the destination folder in the target mailbox as incidents. If the message contains any attachments, they are uploaded to the War Room as files. If the attachment is an email, Cortex XSOAR fetches information about the attached email and downloads all of its attachments (if there are any) as files.
+
+To use Fetch incidents, configure a new instance and select the ```Fetches incidents``` option in the instance settings.
+
+IMPORTANT: The initial fetch interval is the previous 10 minutes. If no emails were fetched before from the destination folder- all emails from 10 minutes prior to the instance configuration and up to the current time will be fetched. Additionally moving messages manually to the destination folder will not trigger fetch incident. Define rules on phishing/target mailbox instead of moving messages manually.
+
+You can configure the ``First fetch timestamp`` field to determine how much time back you want to fetch incidents.
+
+Notice that it might required to set the ``Timeout`` field to a higher value.
+
+Pay special attention to the following fields in the instance settings:
+
+```Email address from which to fetch incidents``` – mailbox to fetch incidents from.
+
+```Name of the folder from which to fetch incidents``` – use this field to configure the destination folder from where emails should be fetched. The default is Inbox folder. Please note, if Exchange is configured with an international flavor `Inbox` will be named according to the configured language.
+
+```Has impersonation rights``` – mark this option if you set the target mailbox to an account different than your personal account. Otherwise Delegation access will be used instead of Impersonation.
```

Comment:
```suggestion
- ```Has impersonation rights``` – mark this option if you set the target mailbox to an account different than your personal account. Otherwise Delegation access will be used instead of Impersonation.
```